tests/Flow/DebounceTaskCancellation.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OwlCoreFlow = OwlCore.Flow;
+
+namespace OwlCore.Tests.Flow
+{
+    [TestClass]
+    public class DebounceTaskCancellation
```

Comment:
Fixed the name of the class and file.

I couldn't find any basic tests for debouncer in the OwlCore also couldn't think of any other tests since there is only 1 method in the whole class. My test validates all possible outcomes of the debouncer.

Reply:
> Fixed the name of the class and file.

The original `DebounceTaskCancellation.cs` file hasn't been removed from the repo.

Reply:
>  Let's rename this class to FlowTests and the file to Flow, to match the convention used by other classes in this folder.

This was also a typo. It should be `Debounce` instead of `Flow` (`public class DebounceTests`, file name `Debounce.cs`)